app/config.go

```diff
@@ -88,12 +88,12 @@ type ServerToml struct {
 // Group holds the Roster and the server-description.
 type Group struct {
 	Roster      *onet.Roster
-	description map[*network.ServerIdentity]string
+	Description map[*network.ServerIdentity]string
 }
 
 // GetDescription returns the description of a ServerIdentity.
```

Comment:
Damn. I could have said a useful comment: 
We should add `this is kept for compatibility reasons for the moment` since basically this method is now completely useless.